backend/rates/history.go

```diff
@@ -49,16 +50,37 @@ var (
 	}
 )
 
+// EnableHistoryPair spins up a new goroutine to periodically update
+// and backfill historical exchange rates.
+// If a pair is unsupported or already enabled, EnableHistoryPair does nothing.
+func (updater *RateUpdater) EnableHistoryPair(coin, fiat string) {
+	coin, fiat = strings.ToUpper(coin), strings.ToUpper(fiat)
```

Comment:
i would remove this, too magical. better to eventually use proper types for coin and fiat.

why not keep them in the original case? BTC will be BTC, and `dai0x6b17` will be `dai0x6b17` instead of `DAI0X6B17`.

Fyi, the coingecko map has wrong entries for dai and sai, should be the above and `sai0x89d2`.

Reply:
it would be nice to use `coin.Code` (as returned by account.Coin().Code()) for the `coin` type. This will save another mapping from coin codes to rates.go codes.

```
coin.CodeBTC:                     "BTC",
			coin.CodeTBTC:                    "BTC",
			coin.CodeLTC:                     "LTC",
			coin.CodeTLTC:                    "LTC",
			coin.CodeETH:                     "ETH",
			coin.CodeRETH:                    "ETH",
			coin.CodeTETH:                    "ETH",
			coin.Code("eth-erc20-usdt"):      "USDT",
			coin.Code("eth-erc20-usdc"):      "USDC",
			coin.Code("eth-erc20-bat"):       "BAT",
			coin.Code("eth-erc20-sai0x89d2"): "SAI0X89D2",
			coin.Code("eth-erc20-dai0x6b17"): "DAI0X6B17",
			coin.Code("eth-erc20-link"):      "LINK",
			coin.Code("eth-erc20-mkr"):       "MKR",
			coin.Code("eth-erc20-zrx"):       "ZRX",
```

(the erc20 codes are from backend/erc20.go).

Reply:
but in the app config.json, they're stored like this:

```json
        "eth": {
            "nodeURL": "etherscan+https://api.etherscan.io/api",
            "transactionsSource": "etherScan",
            "activeERC20Tokens": [
                "usdc",
                "link",
                "sai0x89d2",
                "dai0x6b17"
            ]
        },
```

so, if anything, the map should probably look like:

```go
// A map with values taken from https://api.coingecko.com/api/v3/coins/list,
// representing CoinGecko IDs.
// The keyes must match ethCoinConfig.ActiveERC20Tokens.
var geckoCoin = map[string]string{
	coin.CodeBTC:           "bitcoin",
	coin.CodeLTC:           "litecoin",
	coin.CodeETH:           "ethereum",
	coin.Code("usdt"):      "tether",
	coin.Code("link"):      "chainlink",
	coin.Code("mkr"):       "maker",
	coin.Code("zrx"):       "0x",
	coin.Code("dai0x6b17"): "dai",
	coin.Code("sai0x89d2"): "sai",
	coin.Code("bat"):       "basic-attention-token",
	coin.Code("usdc"):      "usd-coin",
}
```

Anyway, I like the idea of using `coin.Code` instead of a string. Will do.

Reply:
Ok, I've added them with the eth-erc20- prefix but as strings for now. Left coin.Code as a TODO.